esp/Joint/platformio.ini

```diff
@@ -0,0 +1,17 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:esp32doit-devkit-v1]
+platform = espressif32
+board = esp32doit-devkit-v1
+framework = arduino
+upload_port = COM8
```

Comment:
it looks like specification of upload_port makes some troubles - in my case I need `/dev/ttyUSB0` ... but when I used the Communication (?) project this was commented out and autodetect (?) worked fine